app/src/main/java/com/github/sdp_begreen/begreen/models/PhotoMetadata.kt

```diff
@@ -41,4 +42,9 @@ data class PhotoMetadata(var pictureId: String? = null, val title: String? = nul
             return arrayOfNulls(size)
         }
     }
+
+    override fun compareTo(other: PhotoMetadata): Int {
+        return takenOn?.date?.compareTo(other.takenOn?.date) ?: 0
```

Comment:
this is weird, the only thing that separates pictures in general is when they were taken?

Reply:
Yeah that's indeed strictly not right. I assumed that as the dates were containing milliseconds so the probability of a collision was for now pretty low and I was also just trying to program the simplest model working at this stage with paging 3 so that I could totally understand how to implement it with our actual feed.
Thank you for pointing that out.
I'll correct that for the not draft PR.